### PR TITLE
To record the start and end time of job scheduling

### PR DESCRIPTION
**Pull request by dontan001**
This is to fix the issue #2312

### CODE REVIEW

File: pkg/scheduler/metrics/metrics.go [hunk 1]
```diff
@@ -62,6 +62,15 @@ var (
...
 
+	e2eJobSchedulingLastTime = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: VolcanoNamespace,
+			Name:      "e2e_job_scheduling_last_time",
+			Help:      "E2E job scheduling last time",
+		},
+		[]string{"job_name", "queue", "job_namespace"},
+	)
+
```
Comment (hwdef) [inline pkg/scheduler/metrics/metrics.go:73]: This indentation looks weird

Comment (dontan001) [reply pkg/scheduler/metrics/metrics.go:73]: If you expand up this file, it follows the same style as other metrics

**Comment by dontan001** [comment]
> please fix the DCO, LGTM

Fixed, thanks
/assign @hwdef

**Comment by dontan001** [comment]
@hwdef I refined the PR commits to expose the start time metrics as well in Volcano. Thus it is not a must for users to deploy the kube-state-metrics

I also update this in the thread of issue #2312

**Review by hwdef — COMMENTED** [review]
/lgtm

**Comment by hwdef** [comment]
please make CI happy

**Comment by dontan001** [comment]
> please make CI happy

Hi @hwdef , can you please help trigger a retest? since the error in the CI log does not relate to my PR in first glance. 
If the error keeps happening, I can investigate further
Thanks

**Comment by dontan001** [comment]
All the checks passed, please help approve @hwdef @Thor-wl @william-wang

**Review by Thor-wl — APPROVED** [review]

**Comment by dontan001** [comment]
> /approve

Thanks for the help  @Thor-wl @hwdef  @william-wang

**Pull request merged**